### PR TITLE
Check the Source code that have been changed

### PR DESCRIPTION
**Pull request by jayavardhanvny**

### CODE REVIEW

File: src/main/java/org/letuslearn/utils/StringUtility.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+      appendingColumnsWithCommaSeperated.append(iterator.next()).append(",");
+
+    }
+
+    return appendingColumnsWithCommaSeperated;
+  }
+
+  public StringBuffer removeLastChar(StringBuffer sb) {
+    sb.deleteCharAt(sb.length() - 1);
```
Comment (prasadram) [inline src/main/java/org/letuslearn/utils/StringUtility.java:28]: this method is not required here

File: src/main/java/org/letuslearn/utils/StringUtility.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    return sb;
+  }
+
+  public String replaceAtLastChar(StringBuffer sb, String string) {
+
+    sb.replace(sb.length() - 1, sb.length(), string);
```
Comment (prasadram) [inline src/main/java/org/letuslearn/utils/StringUtility.java:34]: this method too

File: src/main/java/org/letuslearn/database/utility/TableMetadataUtility.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+ */
+public class TableMetadataUtility {
+  private List<String> columnsList = new ArrayList<String>();
+
+  public List<String> getColumnsList() {
+    return columnsList;
+  }
+
+  public TableMetadataUtility(ResultSet rscolumns) throws SQLException {
+
```
Comment (prasadram) [inline src/main/java/org/letuslearn/database/utility/TableMetadataUtility.java]: why a utility needs initialization ?

File: src/main/java/org/letuslearn/database/utility/SqlStatement.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+      DatabaseMetaData metadata = connection.getMetaData();
+      rscolumns = metadata.getColumns(connection.getCatalog(), null, tableName, null);
+      TableMetadataUtility tableMetadata = new TableMetadataUtility(rscolumns);
+      columnCount = tableMetadata.getColumnCount(tableName);
+
+      columnsWithCommaSeperated = stringConcatination
+          .getCommaSepratedList(tableMetadata.getColumnsList());
+
+    } catch (SQLException e) {
+
```
Comment (prasadram) [inline src/main/java/org/letuslearn/database/utility/SqlStatement.java]: what we are doing here ?

**Review by prasadram — CHANGES_REQUESTED** [review]

File: src/main/java/org/letuslearn/utils/StringUtility.java (hunk 1 above)
Comment (jayavardhanvny) [reply src/main/java/org/letuslearn/utils/StringUtility.java:28]: not required, but just created for further usage

**Pull request closed without merging**